src/references.d.ts

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
I'm not keen on having our own type declarations, see the comment above about including 3rd party types.